docs/how_to/install.rst

```diff
@@ -1,671 +1,519 @@
-        'djangocms_text_ckeditor': 'djangocms_text_ckeditor.south_migrations',
-    }
-
-Note that older versions of some of the above plugins may use non-standard
-locations for South and Django migrations. Please check each installed plugin
-configuration option to see how to configure Django migrations support.
-
-URL configuration
-=================
+Django Filer
+============
 
-You need to include the ``'cms.urls'`` ``urlpatterns`` **at the end** of your
-``urlpatterns``. We suggest starting with the following
-``~/workspace/myproject/myproject/urls.py``::
+`Django Filer`_ provides file and image management. Many other applications also rely on Django Filer - it's very
+unusual to have a django CMS site that does *not* run Django Filer.
 
-    from django.conf import settings
-    from django.conf.urls import include, url
-    from django.conf.urls.i18n import i18n_patterns
-    from django.conf.urls.static import static
-    from django.contrib import admin
+.. _Django Filer: https://github.com/stefanfoulis/django-filer
 
-    urlpatterns = i18n_patterns('',
-        url(r'^admin/', include(admin.site.urls)),
-        url(r'^', include('cms.urls')),
-    ) + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+To install::
 
+    pip install django-filer
 
-Creating templates
-==================
+A number of applications will be installed as dependencies. `Easy Thumbnails
+<https://github.com/SmileyChris/easy-thumbnails>`_ is required to create new versions of images in different sizes;
+`Django MPTT <https://github.com/django-mptt/django-mptt/>`_ manages the tree structure of the folders in Django Filer.
 
-django CMS uses templates to define how a page should look and what parts of
-it are editable. Editable areas are called **placeholders**. These templates are
-standard Django templates and you may use them as described in the
-`official documentation`_.
+Pillow, the Python imaging library, will be installed. `Pillow <https://github.com/python-imaging/Pillow>`_ needs some
+system-level libraries - the `Pillow documentation <https://pillow.readthedocs.io>`_ describes in detail what is
+required to get this running on various operating systems.
 
-Templates you wish to use on your pages must be declared in the :setting:`CMS_TEMPLATES`
-setting::
+Add::
 
-  CMS_TEMPLATES = (
-      ('template_1.html', 'Template One'),
-      ('template_2.html', 'Template Two'),
-  )
+    'filer',
+    'easy_thumbnails',
+    'mptt',
 
-If you have followed this tutorial from the beginning, this code should
-already be in your settings file.
+to ``INSTALLED_APPS``.
 
-Now, on with the actual template files!
+You also need to add::
 
-Fire up your favourite editor and create a file called ``base.html`` in a folder called
-``templates`` in your ``myproject`` directory.
+    THUMBNAIL_HIGH_RESOLUTION = True
 
-Here is a simple example for a base template called ``base.html``:
+    THUMBNAIL_PROCESSORS = (
```

Comment:
I don't think we should be this detailed about django-filer or other dependency installations.
Instead we should just link to them because otherwise we'd have to maintain it here too and the cms docs shouldn't be affected too much from a change in third party apps.